node/orchestra/README.md

```diff
@@ -88,3 +88,12 @@ As always, debugging is notoriously annoying with bugged proc-macros.
 
 Therefore [`expander`](https://github.com/drahnr/expander) is employed to yield better
 error messages. Enable with `--feature=orchestra/expand`.
+
+## License
+
+Licensed under either of
+
+* Apache License, Version 2.0, (LICENSE-APACHE or <http://www.apache.org/licenses/LICENSE-2.0>)
+* MIT license (LICENSE-MIT or <http://opensource.org/licenses/MIT>)
```

Comment:
MIT or GPLv3? From what I remember, GPLv3 and Apache 2.0 are compatible while Apache 2.0 and MIT has some different wording of the patent claims. Not sure if that's relevant here though.

Reply:
But that's good to know, since our code is GPLv3, so Apache-2.0 is sufficient.